ui/org.eclipse.pde.launching/src/org/eclipse/pde/launching/AbstractPDELaunchConfiguration.java

```diff
@@ -166,7 +169,11 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 		} else {
 			subMonitor.setCanceled(true);
 		}
+	}
 
+	Set<IPluginModelBase> computeLaunchedPlugins(ILaunchConfiguration configuration, IProgressMonitor monitor) throws CoreException {
```

Comment:
I wonder if this should become protected so that sub-classes may override this as well, if they inherit this class directly and don't inherent `EclipseApplicationLaunchConfiguration` or `EquinoxLaunchConfiguration`. But I don't know existing extenders and can therefore not say if that's needed.

Reply:
For now I think we should keep this package-private. If there is a demand for sub-classes to override it, we can still make it protected, but I expect it not to be necessary.